examples/basic/bot.config.yml

```diff
@@ -0,0 +1,5 @@
+verify_token: <verify_token_here>
```

Comment:
As we concluded in #52 that we want to separate configuration from this package, would you mind removing those related-changes from this PR?